docs/docs/gatsby-link.md

```diff
@@ -389,6 +389,16 @@ onClick = () => {
 }
 ```
 
+## Handling stale client-side pages
+
+Gatsby's `<Link>` component will only fetch each page's resources once, when it doesn't have a cached copy. Updates to pages on the site are not reflected in the browser as they are effectively "locked in time". This can have the undesirable impact of different users having different views of the content.
```

Comment:
```suggestion
Gatsby's `<Link>` component will only fetch each page's resources once. Updates to pages on the site are not reflected in the browser as they are effectively "locked in time". This can have the undesirable impact of different users having different views of the content.
```
Wondering if it'd make more sense to add a note about not fetching anything if there is a cached copy?

Reply:
I think it's actually covered by paragraph opening "This only works if the page..."